lib/ansible/modules/web_infrastructure/pmp.py

```diff
@@ -0,0 +1,296 @@
+
+# (c) 2017, Bernat Mut <bernat.mut@aireuropa.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+module: pmp
+version_added: "2.5"
+short_description: Create and modify accounts and groups in Password Manager Pro (PMP)
+description:
+  - Create and modify accounts and groups in Password Manager Pro (PMP).
+
+options:
+  uri:
+    required: true
+    description:
+      - Base URI for the PMP instance.
+  port:
+    required: false
+    default:  7272
+    description:
+      - Base URI for the PMP instance.
+  operation:
+    required: true
+    aliases: [ command ]
+    choices: [ create_host ]
+    description:
+      - The operation to perform.
+
+  token:
+    required: true
+    description:
+      - Token of the API user that performs the action
+
+  group:
+    required: false
+    description:
+      - If exists set the main group of the host.
+
+  resource_name:
+    required: true
+    description:
+     - The name of the resource that will be created.
+
+  account_name:
+    required: true
+    description:
+     - User associated with the created resource.
+
+  password:
+    required: true
+    description:
+     - Password asociated with the user_name.
+
+  owner:
+    required: true
+    description:
+     - Change the default owner, must be an exising user.
+
+  use_proxy:
+    required: false
+    description:
+      - Set if open_url must use proxy
+  validate_certs:
+    required: false
+    description:
+      - Set if open_url must validate certificates
+
+author: "Bernat Mut (@berni69)"
+"""
+
+EXAMPLES = """
+# Retrieve a password:
+- name: Retrieve Password
+  pmp:
+    uri: '{{ pmp_server }}'
+    port: '{{ pmp_port }}'
+    token: '{{ user_token }}'
+    command: 'get_password'
+    resource_name: '{{ server_name }}'
+    account_name: '{{ user_name }}'
+  register: issue
+
+- name: Create server host
+  pmp:
+    uri: '{{ pmp_server }}'
+    port: '{{ pmp_port }}'
+    token: '{{ user_token }}'
+    resource_name: '{{ server_name }}'
+    group: '{{ my_group}}'
+    account_name: '{{ user_name }}'
+    password: '{{ password }}'
+    owner: 'DOMAIN\\USER'
+
+"""
+
+RETURN = '''
+status:
+    description: status of action executed
+    returned: success
+    type: string
+    sample: 'OK'
+password:
+    description: password retrieved
+    returned: success
+    type: string
+    sample: "123456"
+'''
+
+import json
+import sys
+import urllib
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import open_url
+
+
+class PasswordManagerPro:
+    data = {
+        'HOST': '',
+        'PORT': '',
+        'TOKEN': '',
+    }
+
+    use_proxy = False
+    validate_certs = False
```

Comment:
Doesn't really matter since you're not likely to instantiate more than one PasswordManagerPro object in this short script but these should be instance variables, not class attributes.  Move setting of default values into the ```__init__```

Reply:
I'd also default to True for validate_certs as that is a safer default.